sync/src/main/java/com/toggl/sync/processing/ProcessWorkspaces.kt

```diff
@@ -0,0 +1,45 @@
+package com.toggl.sync.processing
+
+import com.toggl.api.models.ApiWorkspace
+import com.toggl.api.models.ApiWorkspaceFeature
+import com.toggl.database.dao.WorkspaceDao
+import com.toggl.database.models.DatabaseWorkspace
+import com.toggl.database.properties.EntitySyncStatus
+import com.toggl.database.properties.StringSyncProperty
+import com.toggl.models.domain.WorkspaceFeature
+
+internal suspend fun processWorkspaces(
```

Comment:
If this is a "free floating function" here, I think the name should mention that it's supposed to be used for processing the workspace after the data was _pulled_ from the server. In Xamarin, I believe this method was under a query specifically intended for saving _pull_ response.
```suggestion
internal suspend fun processPulledWorkspaces(
```